doc/article/en-US/angular1-to-aurelia.md

```diff
@@ -0,0 +1,526 @@
+				...
+			}]);
+		
+		angular.module('myModule')
+			.factory('myFactory, ['$storage', function ($storage) {
+		
+			}]);
+		
+		angular.module('myModule')
+			.value('myValue', "value")
+			... // and so on
+	</source-code>
+</code-listing>
+
+Aurelia handles things a little differently. Instead of framework-specific modules, Aurelia leverages native Javascript modules using either the ES6 import statement or the `require`-style AMD syntax. (These end up transpiled to AMD modules for browser use.) When registering objects with Angular modules Angular made a distinction between `service`, `factory`,  `constant`, and `value` - all syntactic sugar for `provider`. Aurelia, in contrast, makes no special framework distinction between "services", "factories", and other module types - any viewmodels or services are just simple Javascript modules, simplifying the way you can write your code.
+
+So, the angular code above might end up looking more like this:
+
+<code-listing heading="Aurelia Component Definitions">
+	<source-code lang="ES 2015/2016">@inject(Q, HttpClient)
+		export class myService {
+			constructor (Q, http) {
+				...
+			}
+		}
+		
+		@inject(Q, Router)
+		export class anotherService {
+			constructor (Q, router) {
+		
+			}
+		}
+		
+		@inject(Storage)
+		export class myFactory {
+			constructor (storage) {
+		
+			}
+		}
+		
+		export var myValue = value;
+	</source-code>
+</code-listing>
+
+As long as you define your exports semantically, Aurelia doesn't need to register or define them in a specific way ("factory" vs "service" vs "value"). Everything is an exported module; everything is generic, and modules can be imported and then injected using Aurelia's dependency injection.
+
+For more information on dependency injection, see the documentation here.
+
+### Migrating your ViewModels
+
+Fourth, we'll migrate our viewmodels.
+
+In Angular, a controller must be registered with a specific name, using the `angular` global object.
+
+<code-listing heading="Angular ViewModel Definitions">
+	<source-code lang="ES 2015/2016">
+		function Users ($http, $scope) {
+			...
+		}
+		
+		angular.controller('users', ['$scope', '$http', Users]);
+	</source-code>
+</code-listing>
+
+Using Aurelia, any object can act as a viewmodel without being explicitly registered, as long as that object is exported as a module. Essentially, this simply means removing the Angular boilerplate registration code. So, essentially, you would have this:
+
+<code-listing heading="Aurelia ViewModel Definitions">
+	<source-code lang="ES 2015/2016">
+		function Users (httpClient) {
+			...
+		}
+	</source-code>
+</code-listing>
+
+> Note: Dependency Injection
+> Angular was able to inject Angular-specific dependencies based on the argument name (such as `$scope` and `$http`) through its service locator. Because Aurelia does not couple your application to Aurelia-specific implementations, all dependencies must be explicitly injected.
+
```

Comment:
It might be just me, but i'm afraid this example and comment 
> Because Aurelia does not couple your application to Aurelia-specific implementations, all dependencies must be explicitly injected.

doesn't help much to understand how to create viewmodel, that will get its dependencies. I'd add a class with inject annotation instead of this example (as it has a dependency, that needs to be injected).